doc/changes/latest.inc

```diff
@@ -23,12 +23,12 @@ Current (1.6.dev0)
 
 Enhancements
 ~~~~~~~~~~~~
-- None yet
+- Added functionality for linking interactive figures together, such that changing one figure will affect another, see :ref:`tut-ui-events` (:gh:`11685` by `Marijn van Vliet`_)
 
 Bugs
 ~~~~
 - None yet
 
 API changes
 ~~~~~~~~~~~
-- None yet
+- Added :mod:`mne.viz.ui_events` API for controlling interactive figures (:gh:`11685` by `Marijn van Vliet`_).
```

Comment:
This isn't really an API change the way we use it in `latest.inc`... it's a feature. API is really for deprecations